iphone/Classes/TiUIListView.m

```diff
@@ -2188,6 +2196,22 @@ - (void)viewResignFocus
 
 - (void)viewGetFocus
 {
+  if (isSearchBarInNavigation) {
+    id proxy = [(TiViewProxy *)self.proxy parent];
+    while ([proxy isKindOfClass:[TiViewProxy class]] && ![proxy isKindOfClass:[TiWindowProxy class]]) {
+      proxy = [proxy parent];
+    }
+    UIViewController *controller;
+    if ([proxy isKindOfClass:[TiWindowProxy class]]) {
+      controller = [proxy windowHoldingController];
+    } else {
+      controller = [[TiApp app] controller];
+    }
+    if (!controller.navigationItem.searchController) {
+      controller.navigationItem.searchController = searchController;
```

Comment:
I think the iOS 11 guard is required here.

Reply:
It's already taken care while assigning value to  variable 'isSearchBarInNavigation'.
 ` isSearchBarInNavigation = [TiUtils boolValue:[(TiViewProxy *)self.proxy valueForUndefinedKey:@"showSearchBarInNavigation"] def:NO] && [TiUtils isIOSVersionOrGreater:@"11.0"];`

Reply:
@vijaysingh-axway Since we still support older Xcode and SDK versions that don't have the `searchController` property we need the iOS 11 safeguard anyway. Probably something similar to #10729

Reply:
@janvennemann Wrapped under IS_XCODE_9.